.github/workflows/bootstrap.yml

```diff
@@ -13,10 +13,13 @@ jobs:
     env:
       REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL: https://dev-api.ethereum.shapeshift.com
       REACT_APP_UNCHAINED_ETHEREUM_WS_URL: wss://dev-api.ethereum.shapeshift.com
+      REACT_APP_UNCHAINED_AVALANCHE_HTTP_URL: https://dev-api.avalanche.shapeshift.com
+      REACT_APP_UNCHAINED_AVALANCHE_WS_URL: wss://dev-api.avalanche.shapeshift.com
       REACT_APP_UNCHAINED_BITCOIN_HTTP_URL: https://dev-api.bitcoin.shapeshift.com
       REACT_APP_UNCHAINED_BITCOIN_WS_URL: wss://dev-api.bitcoin.shapeshift.com
       REACT_APP_PORTIS_DAPP_ID: 8609e6a8-e0dc-45e6-a0ad-edde63a4cdda
       REACT_APP_ETHEREUM_NODE_URL: https://mainnet.infura.io/v3/d734c7eebcdf400185d7eb67322a7e57
+      REACT_APP_AVALANCHE_NODE_URL: TODO
```

Comment:
`https://api.avax.network/ext/bc/C/rpc` works for now, see https://support.avax.network/en/articles/4626956-how-do-i-set-up-metamask-on-avalanche

Reply:
Nice one, thanks mate - added in https://github.com/shapeshift/web/pull/2105/commits/445563f9184514120e519b27b3eb72c0eac75226

Reply:
probably not a huge deal right now, i will approve also because of api keys and want to make sure how we are managing those or just plain text